codebase/sym.py

```diff
@@ -0,0 +1,30 @@
+import math
+from collections import defaultdict
+
+class Sym():
+
+    def __init__(self, c ,s):
```

Comment:
Set the default values so you need not send it while creating the object.
```suggestion
    def __init__(self, c=0 ,s=""):
```